sentence_transformers/SentenceTransformer.py

```diff
@@ -30,6 +30,27 @@
 
 logger = logging.getLogger(__name__)
 
+
+class HubParameters:
```

Comment:
Consider using a `dataclass` for this :) it will be cleaner and easier to use and to extend in the future. See https://github.com/huggingface/transformers/blob/main/src/transformers/training_args.py#L106 as an example

Reply:
Cool, TIL! Will do 👍